soccer/gameplay/play.py

```diff
@@ -1,5 +1,8 @@
 import composite_behavior
-
+from typing import List
+from situations import Situation
```

Comment:
Should switch to `import situations as situations` if you really want the shortened name